nuxt.config.js

```diff
@@ -98,6 +105,11 @@ export default {
     csv: {}
   },
 
+  sitemap: {
+    hostname: 'https://federalist-edd11e6f-8be2-4dc2-a85e-1782e0bcb08e.app.cloud.gov/site/gsa/usagov-benefits-eligibility/',
```

Comment:
could we get this from federalist ENV vars so that we don't have to change it when we go live?